src/components/Login/Login.js

```diff
@@ -1,96 +1,114 @@
-import React, { Component } from 'react'
-import { login } from './../UserFunctions'
-import './Login.scss';
+import React, { Component } from "react";
+import { login } from "./../UserFunctions";
+import "./Login.scss";
 
 class Login extends Component {
   constructor() {
-    super()
+    super();
     this.state = {
-      email: '',
-      password: '',
-      errors: {}
-    }
+      email: "",
+      password: "",
+      errors: {},
+    };
 
-    this.onChange = this.onChange.bind(this)
-    this.onSubmit = this.onSubmit.bind(this)
+    this.onChange = this.onChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
   }
 
   onChange(e) {
-    this.setState({ [e.target.name]: e.target.value })
+    this.setState({ [e.target.name]: e.target.value });
   }
   onSubmit(e) {
-    e.preventDefault()
+    e.preventDefault();
 
     const user = {
       email: this.state.email,
-      password: this.state.password
-    }
+      password: this.state.password,
+    };
 
-    login(user).then(res => {
+    login(user).then((res) => {
       if (res) {
```

Comment:
We need a check here to see if user is confirmed/active. If not, user should not be able to see his profile and a message "Please wait to get confirmed" should be shown here.